### DISCUSSION TITLE
I've made an XBVR Dark theme

### DISCUSSION THREAD

**Discussion by 7kt-swe** [xbapps/xbvr#1653] [General]
If there people like me who can't look at white pages/interfaces for long without experiencing the sensation of burning eyeballs being stabbed with a fork, I've got some good news for you..

**I've made one and it's available here:** 
[XBVR Dark Edition (greasyfork.org)](https://greasyfork.org/en/scripts/489020-xbvr-dark-edition) userscript
[XBVR Dark Edition (userstyles.world)](https://userstyles.world/style/15250/xbvr-dark-edition) Stylus CSS

*edit*
Added Stylus compatible version

**Comment by vissegard** [comment]
Works great, the only gripe I have is that when you load a tab (like going from scenes to actors) loading is still bright white.

**Reply by 7kt-swe** [reply]
> Works great, the only gripe I have is that when you load a tab (like going from scenes to actors) loading is still bright white.

I'll have to take a look and troubleshoot the problem.
I'll release an update as soon as I find out what's causing it.
May I ask what browser you are using @vissegard ?

**Reply by vissegard** [reply]
I'm using Firefox specifically Floorp fork

**Reply by 7kt-swe** [reply]
okay, do you use the userscript version or the Stylus CSS version? 
I've released an update of the userscript version **1.1.4** (greasyfork) 
Try updating it if you're using that one and get back to me if it solved your issue.

**Reply by vissegard** [reply]
Updated the script, still not resolved. This it how it looks 
![obraz](https://github.com/user-attachments/assets/54d1483d-c721-4316-84a8-2eebec60fb68)

**Reply by 7kt-swe** [reply]
It should be fixed now! update to 1.1.5

**Reply by vissegard** [reply]
Yup, works great. Thank you. Hope it'll make it as a darkmode for the official release.

**Reply by 7kt-swe** [reply]
I really hope so too but I doubt it, I don't even know if they've seen that I've created a dark mode as they haven't posted in the forum thread or gotten in touch
